forge/test/model_demos/high_prio/cnn/pytorch/test_detr.py

```diff
@@ -0,0 +1,69 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+
+# SPDX-License-Identifier: Apache-2.0
+# Detr model having both object detection and segmentation model
+# https://huggingface.co/docs/transformers/en/model_doc/detr
+
+import pytest
+import forge
+import requests
+from torchvision import transforms
+from transformers import (
+    DetrForObjectDetection,
+    DetrImageProcessor,
+    DetrConfig,
+    DetrFeatureExtractor,
+    DetrForSegmentation,
+)
+from PIL import Image
+
+detection_variants = ["facebook/detr-resnet-50"]
```

Comment:
Its okay to just put `"facebook/detr-resnet-50"`  in `@pytest.mark.parametrize` instead of `detection_variants`